### ISSUE TITLE
German TTS: small bug with "ß"

### ISSUE THREAD

**Issue by daganzdaanda** [osmandapp/Osmand#6619]
(osmand 3.2.5) 
I've come across something that might be related to #5954  

When warning of a pedestrian crossing, the TTS voice struggles with the word "Fußgängerübergang".
It comes out as something like "fu gängerübergang", the "ß" is omitted instead of being turned into an "ss" sound. (also, a small break between ...gänger and über... would be good.)

**Comment by sonora** [comment]
Can you please test the following: In OsmAnd, temporarily activate the development plugin. After that, go to Settings / OsnAmd develoment / Test Voice prompts. Select "de-tts", and play test button 9.8. Is the issue present there as well, what do you hear?

Also: Please make sure that in Android (not just OsmAnd), the settings for the TTS engine you are using (there may be several installed) is set to German. Some devices generate funny effects if the language setting in Android and OsmAnd differs.

**Comment by daganzdaanda** [comment]
The phone still runs on Android 4.1.2, the available TTS engines are Google and Samsung. 
The Google engine was active, but it was set to English. Testing with button 9.8 gave the originally described result.  I switched to German, and the result did not change.
With the Samsung engine set on German, the pronunciation was better, the "ß" was not missed.

But then I noticed an update was available for Google Sprachausgabe -- and this update fixed the error! Thanks for pointing me in the right direction!
